<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQuyuzengzhangNormal</class>
 <widget class="QSubDialogBase" name="QQuyuzengzhangNormal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>区域增长normal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>最小簇数量</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>kdtree计算法线K值：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>曲率阀值：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>最大簇数量</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>平滑阀值：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>邻近数量：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSubDialogBase</class>
   <extends>QWidget</extends>
   <header>qsubdialogbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QQuyuzengzhangNormal</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnApply()</slot>
 </slots>
</ui>
